Backend/webblog/accounts/views.py

```diff
@@ -15,8 +15,8 @@ def post(self, request, *args, **kwargs):
         serializer.is_valid(raise_exception=True)
         user = serializer.save()
         return Response({
-                "user": UserSerializer(user, context=self.get_serializer_context()).data,
-                "token": AuthToken.objects.create(user)[1]
+            "user": UserSerializer(user, context=self.get_serializer_context()).data,
+            "token": AuthToken.objects.create(user, )[1]
```

Comment:
Remove this comma.

Reply:
might've been added by the Pycharm.
it's been removed now.